### ISSUE TITLE
Window tiling: Increase/decrease window size behaves incorrectly depending on window selected

### ISSUE THREAD

**Issue by cspipaon** [pop-os/shell#313]
Distribution:
Pop!_OS 20.04 LTS

`apt policy pop-shell`
```
Installed: 0.1.0~1588624471~20.04~dfb378c
Candidate: 0.1.0~1588624471~20.04~dfb378c
```

When having two windows open with tile manager:

1. Stacked side by side: 

- Resizing window on the **right** with Super+Enter -> Ctrl+Left arrow/Right arrow **correctly** decreases/increases of selected window size according to instructions in Keyboard Shortcuts section.

- Resizing window on the **left** with Super+Enter -> Ctrl+Left arrow/Right arrow **incorrectly** decreases/increases window size. Behaviour is the opposite of what is expected to do to the selected window, left arrow increases size, right arrow decreases size.

2. Stacked one on top of another:

- Resizing window on the **top** with Super+Enter -> Ctrl+Up arrow/Down arrow **correctly** increases/decreases window size.

- Resizing window on the **bottom** with Super+Enter -> Ctrl+Up arrow/Down arrow **incorrectly** increases/decreases window size. Behaviour is the opposite of what is expected to do to the selected window, up arrow decreases size, bottom arrow increases size.

----

In other words, when having two windows tiled **horizontally/vertically** it is as if the resizing commands are being applied to the window on the **top/right (respectively)**, disregarding which window is selected.

The same behaviour occurs when having more windows.

**Comment by mmstick** [comment]
Duplicate of #238
